src/main/java/use_cases/add_edit_category_use_case/CategoryOD.java

```diff
@@ -1,19 +1,17 @@
 package use_cases.add_edit_category_use_case;
 
-import entities.Category;
-
 /**
- * Category_Output_Data passes a Category object from programme when the category is successfully added/edited.
+ * Category_Output_Data passes a String message from programme according to the success or fail add/edit category attempt.
  * User will have to re-enter the programme if wish to add or edit a category.(Hence no setCategory())
```

Comment:
This is a little unclear. Maybe you should mention the specific view instead of saying "programme" since the programme refers to the entire project. Also, I don't think you need to specific the setCategory() part.